aot/src/main/java/com/dylibso/chicory/compiler/internal/Methods.java

```diff
@@ -13,9 +13,9 @@
 import com.dylibso.chicory.wasm.InvalidException;
 import com.dylibso.chicory.wasm.types.FunctionType;
 
-public final class AotMethods {
+public final class Methods {
```

Comment:
This looks a bit too generic ... the meaning would be: "Methods available to use by the generated Java bytecode".
Not sure how to capture it ...

Reply:
GeneratedMethods, TemplateMethods?

Reply:
make it GeneratedMethods now.